quantumdb-cli/src/main/java/io/quantumdb/cli/commands/Init.java

```diff
@@ -33,7 +33,7 @@ public void perform(CliWriter writer, List<String> arguments) {
 				String user = getArgument(arguments, "username", String.class);
 				String pass = getArgument(arguments, "password", String.class, null);
 
-				config.setUrl("jdbc:postgresql://" + hosts + "/" + catalogName + "?targetServerType=master\"");
+				config.setUrl("jdbc:postgresql://" + hosts);
```

Comment:
The command (as described in the docs) is: `init --host=localhost:5432 --database=<database> --username=<username> --password=<password>`. With your change this would result into: `jdbc:postgresql://localhost:5432` which isn't correct:
- You still need to select the specified database. The driver might default to a database with the same name as the user, but we need the database name anyway.
- In case the user is using a primary/follower setup, you'll want to ensure the primary is targeted. In that case the command will look like `init --host=server1:5432,server2:5432 --database=<database> --username=<username> --password=<password>` and should yield the url: `jdbc:postgresql://server1:5432,server2:5432/<database>?targetServerType=primary`.

```suggestion
				config.setUrl("jdbc:postgresql://" + hosts + "/" + catalogName + "?targetServerType=primary\"");
```

Reply:
The database name should be either added here or in the change below, not at both places.
Additionally, in your suggestion there is a \" which has no starting \"

Reply:
Ah I see what you mean now. Not sure how that one slipped in. So two options in this case:

- Let every caller create their own URL and use that.
- Construct the URL once here in `Init.java`, and reuse that everywhere (currently `Query.java:171`, and `PostgresqlBackend.java:90`).

Seeing as all callers should be using the same URL, I think the latter option would be best.